### PR TITLE
+Add APIObject

### PR DESCRIPTION
**Pull request by wongpratan**

### CODE REVIEW

File: AppBuilder/ABFactory.js [hunk 1]
```diff
@@ -39,10 +39,10 @@ import Tenant from "../resources/Tenant.js";
 import UISettings from "./uiSettings/config.js";
 // UISettings: detailed settings for our common UI elements
 
-import Webix from "../js/webix/webix.min.js";
+// import Webix from "../js/webix/webix.min.js";
 // NOTE: moved to require() because using import with webix.js
 // really messed things up!
-// var Webix = require("../js/webix/webix.js");
+var Webix = require("../js/webix/webix.js");
```
Comment (nh758) [inline AppBuilder/ABFactory.js:45]: I don't understand why we would need to change this now?  We've been using import for about a year already.

File: AppBuilder/platform/ABModelApi.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      return Promise.reject(error);
+   }
+
+   /**
+    * @method create
+    * update model values on the server.
+    */
+   create(values) {
+      var error = new Error(
+         "ABObjectQuery.ABModelAPI.create() should not be called."
```
Comment (nh758) [inline AppBuilder/platform/ABModelApi.js:30]: Long term, I think we will implement some of these methods. Maybe a better warning is "not implemented."

**Review by nh758 — APPROVED** [review]

File: AppBuilder/ABFactory.js (hunk 1 above)
Comment (wongpratan) [reply AppBuilder/ABFactory.js:45]: I'm sorry. I did not clean up the code. 

I uses it for debugging.

**Review by guyyoo — APPROVED** [review]
LGTM.

**Pull request merged**